### PR TITLE
ethereumjs-testing PR #38 branch integration test

### PR DESCRIPTION
**Pull request by holgerd77**
Short integration test for the updated ``getTests()`` function in  https://github.com/ethereumjs/ethereumjs-testing/pull/38 worked on by @whymarrh.

DO NOT MERGE!

### CODE REVIEW

**Comment by whymarrh** [comment]
@holgerd77 it looks like 0e6315d (which uses a older pre-change commit hash) is also causing the coverage to decrease. This might be caused by something else.

File: karma.conf.js [hunk 1]
```diff
@@ -18,7 +18,6 @@ module.exports = function (config) {
 
     // list of files / patterns to exclude
     exclude: [
-      './tests/api/state/stateManager.js' // 4, "# should clear the cache when the state root is set"
```
Comment (whymarrh) [inline karma.conf.js]: Why is this removed?

Comment (holgerd77) [reply karma.conf.js]: This was just a test, since the reason for your testing PR is to get these browser tests to pass (at least asyncawait dependency was one is eventual several blockers). Coverage decrease is also happening without this commit.

**Comment by holgerd77** [comment]
One reason for failure (maybe together with the testing PR changes) is that coverage test execution script in package.json slightly differs from the direct test execution scripts (different parameters). 

There is nevertheless likely also an underlying issue, eventually the usage of the default fork setting in tester.js.

**Comment by holgerd77** [comment]
So, "direct" means e.g. "testStatePetersburg" also in package.json.

**Comment by whymarrh** [comment]
> One reason for failure (maybe together with the testing PR changes)

I don't understand, sorry. The latest commit on this PR uses a version of ethereumjs-testing that has the old `getTests` function (i.e., no changes whatsoever). I don't understand how this might be related to the changes in ethereumjs-testing.

**Comment by holgerd77** [comment]
@whymarrh Got it, your [fork](https://github.com/whymarrh/ethereumjs-testing#cleanup-getTests) is pretty outdated and therefore and old submodule ``tests`` folder was used which didn't include the proper ``Constantinople``/``Petersburg`` tests yet (I would generally still prefer if you could do work directly on the original libraries instead of your own fork, maybe you can switch habits here?).

So we should be save to do the release, now ran the tests against the already merged ``master`` of ``ethereumjs-testing``, coverage stays the same. 😄

**Comment by holgerd77** [comment]
Just for info, not directly related to this PR:

Have set the following tests to now be required:

- ``test_api_browser``
- ``test_blockchain``
- ``test_state_petersburg``

Let me know if you have any comments on that.

**Comment by holgerd77** [comment]
Functionality integrated on the `ethereumjs-testing` side, will close.
